external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/AbstractHBaseBolt.java

```diff
@@ -51,11 +51,11 @@ public AbstractHBaseBolt(String tableName, HBaseMapper mapper) {
     }
 
     @Override
-    public void prepare(Map map, TopologyContext topologyContext, OutputCollector collector) {
+    public void prepare(Map topoConf, TopologyContext topologyContext, OutputCollector collector) {
         this.collector = collector;
         final Configuration hbConfig = HBaseConfiguration.create();
 
-        Map<String, Object> conf = (Map<String, Object>)map.get(this.configKey);
+        Map<String, Object> conf = (Map<String, Object>)topoConf.get(this.configKey);
```

Comment:
No because of the order of operations.  It is casting the value returned by `topoConf.get`, not `topoConf` itself.